documentation/tutorial.md

```diff
@@ -1,7 +1,14 @@
-The following tutorial is an accurate representation of the experimental OpenSearch Flow OSD Plugin as of 11/18/2024, based on OSD 2.18.
+The following tutorial is an accurate representation of the experimental OpenSearch Flow OSD Plugin as of 11/27/2024, based on OSD 2.18.
+
+Changelog:
```

Comment:
@ohltyler, I feel having Changelog.md file to track all the changes would be better than updating Changelog here. 
- so that we can also update it for every PR. 
- Example: https://github.com/opensearch-project/opensearch-py/blob/main/CHANGELOG.md

Reply:
I am fine if you would like to do it in followup PR.

Reply:
This isn't supposed to be a repo-level changelog, just a small list of tracked changes to the tutorial itself.